durabletask/internal/shared.py

```diff
@@ -10,6 +10,7 @@
 import grpc
 
 from durabletask.internal.grpc_interceptor import DefaultClientInterceptorImpl
+from externalpackages.durabletaskscheduler.durabletask_grpc_interceptor import DTSDefaultClientInterceptorImpl
```

Comment:
We should not have DTS-specific code in the `durabletask` library. If we need new abstractions in the `durabletask` library, then we need to make them generic in such a way that they can be used for DTS or for non-DTS scenarios.